samples/backchannel/index.html

```diff
@@ -21,74 +21,80 @@
-	<script>
-            var params = BotChat.queryParams(location.search);
+      .example h2 {
+        font-family: 'Segoe UI';
+      }
 
-            var user = {
-                id: params['userid'] || 'userid',
-                name: params["username"] || 'username'
-                };
-            
-            var bot = {
-                id: params['botid'] || 'botid',
-                name: params["botname"] || 'botname'
-            };
+      #BotChatGoesHere {
+        border: 1px solid #333;
+        float: left;
+        height: 600px;
+        position: relative;
+        width: 460px;
+      }
+    </style>
+  </head>
+  <body>
+    <section class="example">
+      <h2>Type a color into the WebChat!</h2>
+      <button onclick="postButtonMessage()" style="height: 60px; margin-left: 80px; margin-top: 20px; padding: 20px; width: 120px;">Click Me!</button>
+    </section>
+    <div id="BotChatGoesHere"></div>
+    <script src="../../botchat.js"></script>
+    <script>
+      const params = BotChat.queryParams(location.search);
 
-            window['botchatDebug'] = params['debug'] && params['debug'] === "true";
+      const user = {
+        id: params['userid'] || 'userid',
+        name: params['username'] || 'username'
+      };
 
-            var botConnection = new BotChat.DirectLine({
-                secret: params['s'],
-                token: params['t'],
-                domain: params['domain'],
-                webSocket: params['webSocket'] && params['webSocket'] === "true" // defaults to true
-            });
+      const bot = {
+        id: params['botid'] || 'botid',
+        name: params['botname'] || 'botname'
+      };
 
-            BotChat.App({
-                botConnection: botConnection,
-                user: user,
-                bot: bot
-            }, document.getElementById("BotChatGoesHere"));
+      window['botchatDebug'] = params['debug'] && params['debug'] === 'true';
 
-            botConnection.activity$
-                .filter(activity => activity.type === "event" && activity.name === "changeBackground")
-                .subscribe(activity => changeBackgroundColor(activity.value))
+      const botConnection = new BotChat.DirectLine({
+        domain: params['domain'],
+        secret: params['s'],
+        token: params['t'],
+        webSocket: params['webSocket'] && params['webSocket'] === 'true' // defaults to true
+      });
 
-            const changeBackgroundColor = (newColor) => {
-                document.body.style.backgroundColor = newColor;
-            }
+      BotChat.App({ bot, botConnection, user }, document.getElementById('BotChatGoesHere'));
 
-            const postButtonMessage = () => {
-                botConnection
-                    .postActivity({type: "event", value: "", from: {id: "me" }, name: "buttonClicked"})
-                    .subscribe(id => console.log("success"));
-            }
+      const changeBackgroundColor = newColor => {
```

Comment:
Noticed `/backchannel` samples right now also using arrow function, will move them to traditional one too.